lib/scripts/services/contextMenu.js

```diff
@@ -17,6 +17,20 @@ export class ContextMenu {
     });
   }
 
+  searchForWordUsageExamples(selectedText, searchPlatform ) {
+    var searchPlatformURLS = {
+      'yourdictionary.com': 'http://sentence.yourdictionary.com/%s',
+      'use-in-a-sentence.com': 'http://www.use-in-a-sentence.com/english-words/10000-words/%s.htm',
+      'manythings.org': 'http://www.manythings.org/sentences/words/%s/1.html'
+    }
+    var searchUrl = searchPlatformURLS[searchPlatform];
+    selectedText = selectedText.replace(" ", "+");
+    searchUrl = searchUrl.replace(/%s/g, selectedText);
+    chrome.tabs.create({
```

Comment:
@ankit-m this shows that a new tab is created... I think this answers your question.